orm/src/test/java/org/hibernate/search/test/query/dsl/DSLTest.java

```diff
@@ -890,6 +890,57 @@ public void testUseMatchQueryOnEmbeddedNumericIdCreatesTermQuery() throws Except
 		);
 	}
 
+	@Test
```

Comment:
For my taste `DSLTest` gets a big too big to be easily understandable. It gets quite hard to figure out which test classes are involved where. Why not create a single test class for these criteria issue, maybe even in a dedicated directory.